src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -1227,7 +1231,8 @@ private void updateToolFlag(@NotNull final String tool, final int required)
                 checkForPickaxe(required);
                 break;
             default:
-                Log.getLogger().error("Invalid tool " + tool + " not implemented as tool!");
+                checkForPickaxe(DIAMOND_LEVEL);
```

Comment:
Aren't we requesting tools from our codebase? So shouldn't we just have all tools implemented?